server/database/data/build.js

```diff
@@ -0,0 +1,9 @@
+const { readFileSync } = require('fs');
+const { join } = require('path');
+const connection = require('./connection');
+
+const bulidDb = () => {
+  const sql = readFileSync(join(__dirname, 'rawdaty.sql')).toString();
```

Comment:
Here we need run the fuke data when test
```suggestion
  const sql = readFileSync(join(__dirname, 'rawdaty.sql')).toString();
  if (process.env.NODE_ENV === 'test') {
    sql += readFileSync(join(__dirname, 'rawdatyFake.sql')).toString();
  }
```

Reply:
good job Osama but the `sql` should be declared with `let`